### PR TITLE
Ragini/public page powered by/test

### PR DESCRIPTION
**Pull request by ViaSocket-Git**

### CODE REVIEW

File: src/components/apiDocReview/apiDocReview.scss [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
-  color:#adadad;
-  gap: 20px;
+.feedback-saved {
+  text-align: center;
+  color: green;
+  font-weight: light;
 }
+
+.enpoint-title {
+  > :first-child {
```
Comment (Saurabh186) [inline src/components/apiDocReview/apiDocReview.scss:22]:
```suggestion
   &:first-child {
```aise hoga yeh

File: src/components/main/main.scss [hunk 2]
```diff
@@ -141,11 +141,17 @@
...
-html body {
-  background-color: #fff;
-  font-family: 'Inter', sans-serif;
-  font-size: $font-normal;
-  color: $text-color;
+html {
+  scrollbar-width: thin;
+
+  body {
+    background-color: #fff;
```
Comment (Saurabh186) [inline src/components/main/main.scss:148]: iska variables tumne banaya hai toh use bhi kro

File: src/components/main/poweredByTechdoc/poweredByTechdoc.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.footerWrapper {
+    border-top: 0;
```
Comment (Saurabh186) [inline src/components/main/poweredByTechdoc/poweredByTechdoc.scss:2]: use bootstrap class

File: src/components/main/poweredByTechdoc/poweredByTechdoc.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+    .confetti-wrap {
+        user-select: none;
+
+        .fetti {
```
Comment (Saurabh186) [inline src/components/main/poweredByTechdoc/poweredByTechdoc.scss:7]: kya nam hai yar fetti??
run time me ban ra hai kya??
run time me nahi ban ra ho border radius bootstrap class use aur 3 4 se jada hai toh yhi likhe rahne dena

File: src/components/main/poweredByTechdoc/poweredByTechdoc.scss [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+            position: fixed;
+            left: var(--confetti-x-position);
+            top: var(--confetti-y-position);
+            width: 10px;
+            height: 10px;
+            border-radius: 50%;
+        }
+
+        > :first-child {
```
Comment (Saurabh186) [inline src/components/main/poweredByTechdoc/poweredByTechdoc.scss:16]:
```suggestion
         &:first-child {
```

File: src/components/main/poweredByTechdoc/poweredByTechdoc.scss [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+            > :first-child {
```
Comment (Saurabh186) [inline src/components/main/poweredByTechdoc/poweredByTechdoc.scss:17]:
```suggestion
            &:first-child {
```

**Review by Saurabh186 — CHANGES_REQUESTED** [review]

File: src/components/apiDocReview/apiDocReview.jsx [hunk 7]
```diff
@@ -205,37 +205,25 @@ const ApiDocReview = (props) => {
...
-                onClick={() => {
-                  handleFeedback('DISLIKE')
-                }}
-              >
-                {showSolidDislike ? <BiSolidDislike size={20} /> : <BiDislike size={20} />}
-              </div>
-            </OverlayTrigger>
+      <div className='position-relative d-flex m-auto flex-column enpoint-title'>
+        <div className='d-inline-flex justify-content-around align-items-center border p-3'>
+          <span className='d-flex mx-2 justify-content-center font-14'>Was this page helpful?</span>
```
Comment (Saurabh186) [inline src/components/apiDocReview/apiDocReview.jsx:210]: remove mx-2 and use gap- 2 in parant

File: src/components/apiDocReview/apiDocReview.jsx [hunk 8]
```diff
@@ -205,37 +205,25 @@ const ApiDocReview = (props) => {
...
+          <div className='cursor-pointer mx-2 font-12 d-flex align-items-center like-btn' onClick={() => { handleLikeButton() }}>
```
Comment (Saurabh186) [inline src/components/apiDocReview/apiDocReview.jsx:211]: remove

File: src/components/apiDocReview/apiDocReview.jsx [hunk 9]
```diff
@@ -205,37 +205,25 @@ const ApiDocReview = (props) => {
...
+            <div className='mx-1'>
+              {liked ? <BiSolidLike size={20} /> : <BiLike size={20} />}
+            </div>
+            <span>Yes</span>
           </div>
+          <div className='cursor-pointer mx-2 font-12 d-flex align-items-center dislike-btn' onClick={() => { handleFeedback('DISLIKE') }}>
```
Comment (Saurabh186) [inline src/components/apiDocReview/apiDocReview.jsx:217]: remove mx

**Review by Saurabh186 — CHANGES_REQUESTED** [review]

File: src/components/apiDocReview/apiDocReview.scss (hunk 1 above)
Comment (ViaSocket-Git) [reply src/components/apiDocReview/apiDocReview.scss:22]: Resolved

File: src/components/main/poweredByTechdoc/poweredByTechdoc.scss (hunk 3 above)
Comment (ViaSocket-Git) [reply src/components/main/poweredByTechdoc/poweredByTechdoc.scss:2]: Resolved

File: src/components/main/poweredByTechdoc/poweredByTechdoc.scss (hunk 4 above)
Comment (ViaSocket-Git) [reply src/components/main/poweredByTechdoc/poweredByTechdoc.scss:7]: run time

File: src/components/main/poweredByTechdoc/poweredByTechdoc.scss (hunk 6 above)
Comment (ViaSocket-Git) [reply src/components/main/poweredByTechdoc/poweredByTechdoc.scss:17]: Resoved

File: src/components/apiDocReview/apiDocReview.jsx (hunk 7 above)
Comment (ViaSocket-Git) [reply src/components/apiDocReview/apiDocReview.jsx:210]: Resolved

File: src/components/apiDocReview/apiDocReview.jsx (hunk 9 above)
Comment (ViaSocket-Git) [reply src/components/apiDocReview/apiDocReview.jsx:217]: Resolved

File: src/components/apiDocReview/apiDocReview.jsx (hunk 8 above)
Comment (ViaSocket-Git) [reply src/components/apiDocReview/apiDocReview.jsx:211]: Resolved

File: src/components/main/poweredByTechdoc/poweredByTechdoc.scss (hunk 5 above)
Comment (ViaSocket-Git) [reply src/components/main/poweredByTechdoc/poweredByTechdoc.scss:16]: Resolved

File: src/components/main/main.scss (hunk 2 above)
Comment (ViaSocket-Git) [reply src/components/main/main.scss:148]: Resolved

**Review by Saurabh186 — APPROVED** [review]

**Pull request closed without merging**